Lib/trufont/windows/fontWindow.py

```diff
@@ -1048,6 +1069,29 @@ def _proceedWithDeletion(self, erase=False):
                 return True
             return False
 
+    def _svg2glif(self, svg, name, width=0, height=0, unicodes=None, transform=None,
+                version=2):
+        """ Convert an SVG outline to a UFO glyph with given 'name', advance
+        'width' and 'height' (int), and 'unicodes' (list of int).
+        Return the resulting string in GLIF format (default: version 2).
+        If 'transform' is provided, apply a transformation matrix before the
+        conversion (must be tuple of 6 floats, or a FontTools Transform object).
+        """
+        glyph = SimpleNamespace(width=width, height=height, unicodes=unicodes)
+        outline = SVGPath.fromstring(svg, transform=transform)
+
+        # writeGlyphToString takes a callable (usually a glyph's drawPoints
+        # method) that accepts a PointPen, however SVGPath currently only has
+        # a draw method that accepts a segment pen. We need to wrap the call
+        # with a converter pen.
+        def drawPoints(pointPen):
+            pen = SegmentToPointPen(pointPen)
+            outline.draw(pen)
+
+        return writeGlyphToString(name,
+            glyphObject=glyph,
+            drawPointsFunc=drawPoints,
+            formatVersion=version)
```

Comment:
I'm not sure why you write to a glyph then read from string right after, also why creating another glyph object here when the caller already has glyph/otherGlyph. I think even otherGlyph can be removed in this instance as the SVGPath object can draw itself into a pen.

So you can do svg.draw(glyph.getPen()). Anyway, I'll do the merge. Thanks!